workers/unity/Packages/io.improbable.gdk.gameobjectcreation/LinkedGameObjectMapSubscriptionManager.cs

```diff
@@ -17,15 +16,12 @@ public LinkedGameObjectMapSubscriptionManager(World world)
 
         public override Subscription<LinkedGameObjectMap> Subscribe(EntityId entityId)
         {
-            if (linkedGameObjectMapSubscription == null)
-            {
-                linkedGameObjectMapSubscription = new Subscription<LinkedGameObjectMap>(this, new EntityId(0));
+            var linkedGameObjectMapSubscription = new Subscription<LinkedGameObjectMap>(this, new EntityId(0));
```

Comment:
random point, but considering we use `new EntityId(0)` in a few places, should we include a built-in `EntityId.zero`?

Reply:
Tbh I don't think so - the utility to users is probably pretty low.